### DISCUSSION TITLE
I am new in development field,Can I contribute in this project

### DISCUSSION THREAD

**Discussion by gigler** [Daksh777/TakeABreak#12] [Q&A]
I want to contribute in this project bcz i think this project will help me in my personal use.
I do not know any tech used in it.Can you help me or guide in this so that I learn this tech for build or fixing some bug in this project.
Thanks

**Comment by Daksh777** [comment]
Hey @gigler 
You need to have at least the basic knowledge of JavaScript, CSS and HTML to be able to contribute to the project otherwise you won't be able to contribute.
